src/Cache/MongoLock.php

```diff
@@ -14,20 +16,18 @@ final class MongoLock extends Lock
     /**
      * Create a new lock instance.
      *
-     * @param Collection  $collection              The MongoDB collection
-     * @param string      $name                    Name of the lock
-     * @param int         $seconds                 Time-to-live of the lock in seconds
-     * @param string|null $owner                   A unique string that identifies the owner. Random if not set
-     * @param array       $lottery                 The prune probability odds
-     * @param int         $defaultTimeoutInSeconds The default number of seconds that a lock should be held
+     * @param Collection      $collection The MongoDB collection
+     * @param string          $name       Name of the lock
+     * @param int             $seconds    Time-to-live of the lock in seconds
+     * @param string|null     $owner      A unique string that identifies the owner. Random if not set
+     * @param array{int, int} $lottery    Probability [chance, total] of pruning expired cache items. Set to [0, 0] to disable
      */
     public function __construct(
         private readonly Collection $collection,
         string $name,
         int $seconds,
         ?string $owner = null,
         private readonly array $lottery = [2, 100],
-        private readonly int $defaultTimeoutInSeconds = 86400,
```

Comment:
Why was this removed entirely? It looks like it previously served a purpose if `$seconds` was zero. Now, specifying zero for `$seconds` would cause locks to immediately expire.